### DISCUSSION TITLE
Welcome to inputmodule-rs Discussions!

### DISCUSSION THREAD

**Discussion by JohnAZoidberg** [FrameworkComputer/inputmodule-rs#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jannic** [comment]
Hi!

As an active member of the embedded rust community, I am pleased to see that you are using rust for some of the input modules.
Of course, I tried to build the code, but failed because some crate is not public:
```
Unable to update ssh://git@github.com/FrameworkComputer/st7306.git
```
Is that just an oversight or is that crate not ready for publication?

If you have questions regarding `rp2040-hal` and related crates, join us at https://matrix.to/#/#rp-rs:matrix.org. Any feedback is welcome, as well.

**Reply by jannic** [reply]
The st7306 is public now, and the build works. Thank you!
